api/classes/class-lightweight-api.php

```diff
@@ -231,26 +233,19 @@ public function get_transient( $name ) {
 		global $wpdb;
 		$name = '_transient_' . $name;
 
-		$empty_transients = wp_cache_get( 'empty_transients', 'newspack-popups' );
-
-		if ( ! is_array( $empty_transients ) ) {
-			$empty_transients = array();
-		}
-
-		if ( isset( $empty_transients[ $name ] ) ) {
+		$value = wp_cache_get( $name, 'newspack-popups' );
+		if ( -1 === $value ) {
+			$this->debug['read_empty_transients'] += 1;
 			$this->debug['cache_count'] += 1;
 			return null;
-		}
-
-		$value = wp_cache_get( $name, 'newspack-popups' );
-		if ( false === $value ) {
+		} elseif ( false === $value ) {
 			$this->debug['read_query_count'] += 1;
 			$value = $wpdb->get_var( $wpdb->prepare( "SELECT option_value FROM $wpdb->options WHERE option_name = %s LIMIT 1", $name ) ); // phpcs:ignore
 			if ( $value ) {
-				wp_cache_add( $name, $value );
+				wp_cache_set( $name, $value );
```

Comment:
Do we want to use the Group Name param, too?
```
wp_cache_set( $name, $value, 'newspack-popups' );
```

Reply:
Eek, you're absolutely right. Fixed inhttps://github.com/Automattic/newspack-popups/pull/208/commits/7b229574adcefb5a578980b254b0cf294a283e61